src-input/duk_bi_json.c

```diff
@@ -1423,7 +1423,11 @@ DUK_LOCAL void duk__enc_fastint_tval(duk_json_enc_ctx *js_ctx, duk_tval *tv) {
 	 * "long long" type exists.  Could also rely on C99 directly but that
 	 * won't work for older MSVC.
 	 */
+#if defined(__MINGW32__) || defined(__MINGW64__)
+	DUK_SPRINTF((char *) buf, "%I64d", (long long) v);
+#else
 	DUK_SPRINTF((char *) buf, "%lld", (long long) v);
+#endif
```

Comment:
@beevik Yeah, I was thinking along the lines of `DUK_FMT_LONGLONG` or something. It's unfortunately compiler/platform (or more strictly speaking, lib) specific but as long as it can be defined to match the sprintf() provider it should be OK.